versioned_docs/version-2020fa/assignment1.md

```diff
@@ -20,7 +20,7 @@ yarn add --dev typescript ts-node @types/express
 ```
 
 :::info
-Check out [lecture1](/docs/lecture1) for more details on how this setup works!
+Check out [lecture1](/docs/2020fa/lecture1) for more details on how this setup works!
```

Comment:
is this supposed to be 2022sp or this one doesn't matter?

Reply:
This was in 2020fa so i just added it. Not sure why it wasnt updated before.